src/main/java/net/discordjug/javabot/util/WebhookUtil.java

```diff
@@ -157,8 +167,17 @@ private static <T, R> R transformOrNull(T toTransform, Function<T, R> transforme
 	 * @param threadId          id of the thread in which the message should be replaced
 	 * @param embeds            optional additional embeds to be added
 	 */
+	public static void replaceMemberMessage(Webhook webhook, Message originalMessage, String newMessageContent, long threadId, List<Message.Attachment> attachments, List<MessageEmbed> embeds) {
+		WebhookUtil.mirrorMessageToWebhook(webhook, originalMessage, newMessageContent, threadId, null, attachments, embeds)
+				.thenAccept(unused -> originalMessage.delete().queue())
+				.exceptionally(e -> {
+					ExceptionLogger.capture(e, WebhookUtil.class.getSimpleName());
+					return null;
+				});
+	}
+
 	public static void replaceMemberMessage(Webhook webhook, Message originalMessage, String newMessageContent, long threadId, MessageEmbed... embeds) {
```

Comment:
Again, please reduce unnecessary code and just replace the message body with the overload:
```java
public static void replaceMemberMessage(Webhook webhook, Message originalMessage, String newMessageContent, long threadId, MessageEmbed... embeds) {
	replaceMemberMessage(webhook, originalMessage, newMessageContent, threadId, null, List.of(embeds))
```